docs/source/get_started/access_info/access_info_qss.ipynb

```diff
@@ -0,0 +1,307 @@
+{
```

Comment:
<p>nit: SuperstaQ -&gt; Superstaq.</p><p> </p><p>(Also, just a note, after PR #468, <code>status()</code> would be able to give the following <code>qiskit.providers.jobstatus</code></p><p>: <code>JobStatus.QUEUED</code>, <code>JobStatus.RUNNING</code>, <code>JobStatus.INITIALIZING</code>, <code>JobStatus.CANCELLED</code>,<code>JobStatus.ERROR</code>, <code>JobStatus.DONE</code>... But maybe this doesn't have to be updated now.)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SupertechLabs/client-superstaq/pull/543/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>